localstack-core/localstack/services/apigateway/next_gen/execute_api/test_invoke.py

```diff
@@ -140,9 +179,15 @@ def create_test_invocation_context(
         # TODO: handle multiValueHeaders
         body=to_bytes(test_request.get("body") or ""),
     )
+
     invocation_context.invocation_request = invocation_request
+    try:
+        # this is AWS behavior, it will accept any value for the `pathWithQueryString`, even if it doesn't match
+        # the expected format. It will just fall back to no value if it cannot parse the path parameters out of it
+        _, path_parameters = RestAPIResourceRouter(deployment).match(invocation_context)
+    except Exception:
+        path_parameters = {}
```

Comment:
this is also part of the fix, due to the ability to pass invalid values, but it was not strictly needed for the sample to work